modules/integration/test-results-service/dependency-reduced-pom.xml

```diff
@@ -4,15 +4,14 @@
     <artifactId>msf4j-service</artifactId>
     <groupId>org.wso2.msf4j</groupId>
     <version>2.1.0</version>
-    <relativePath>../pom.xml/pom.xml</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.wso2.sp.example</groupId>
+  <groupId>org.wso2.sp.events</groupId>

```

Comment:
We need to stick to the same group id as the integration pom which should be the parent.